app/javascript/controllers/hide_by_source_val_controller.js

```diff
@@ -2,13 +2,24 @@ import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
   static targets = ["source", "destination"]
-  static values = {
-    valuesToHide: Array
+  static values = { valuesToShow: Array }

```

Comment:
Wouldn't this break existing code? We should support both "hide" and "show" here depending on which one we passed in.